scripts/quickstart.py

```diff
@@ -9,11 +9,18 @@
 from dask.distributed import Client
 
 SOFTWARE = os.environ["SOFTWARE_ENV"]
+storage_options = {
```

Comment:
@rrpelgrim can we remove this, as we are not using it?

Reply:
Yes, good catch!